index.html

```diff
@@ -1,44 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Flick-fil-a</title>
-  </head>
-  <body>
-    <div id="app-container">
-      <!-- logo -->
-      <div class="logo">
+</head>
+<body>
+<div id="app-container">
+    <!-- logo -->
+    <div class="logo">
         <div class="container">
-          <p><span>Flick</span>-fil-<span>a</span></p>
+            <p><span>Flick</span>-fil-<span>a</span></p>
         </div>
-      </div>
-      <!-- end of logo -->
+    </div>
+    <!-- end of logo -->
 
-      <!-- Search container -->
-      <div id="search-container">
+    <!-- Search container -->
+    <div id="search-container">
         <div class="search-element">
-          <input
-            type="text"
-            class="form-control"
-            placeholder="Search Movie Title ..."
-            id="movie-search-box"
-          />
+            <input
+                    type="text"
+                    class="form-control"
+                    placeholder="Search Movie Title ..."
+                    id="movie-search-box"
+            />
+            <label for="movie-search-box"></label>
```

Comment:
why is there an empty label? my guess is this was added to make a linter happy?

Reply:
Yes, it was, but I actually wanted to leave it out.

Reply:
This is the oldest HTML issue in the history of time. Browsers & accessibility tools (like screen readers) are designed with the assumption that there will be a label element for every input, but UX design often omits visible labels for style reasons (even https://www.google.com/ does this), and so developers and users get caught in the middle with no great option. 

There's a lot of issues with not having a label:
* less space to click on 
* issues for screen reader users
* some people struggle or get confused

Checkout https://stackoverflow.com/questions/64167749/what-to-do-if-i-dont-have-a-label-element-to-go-with-an-input-element and https://beingfrankly.nl/blog/accessible-input-without-label/ to get some rough background (I don't know if either of these are accurate in 2023). If you don't want to add a label, consider using the `aria-label` attribute as a good compromise (I think this is better than making an invisible label). I wouldn't stress about it in this project but it's definitely a good issue to be aware of in the web space. I could see it as an interview question.